src/Integration.Vsix/ErrorList/IssuesSnapshot.cs

```diff
@@ -181,14 +181,22 @@ public override bool TryGetValue(int index, string keyName, out object content)
                     // Note: the line and column numbers are taken from the SnapshotSpan, not the Issue.
                     // The SnapshotSpan represents the live document, so the text positions could have
                     // changed from those reported from the Issue.
-                    content = issueViz.Span.Value.Start.GetContainingLine().LineNumber;
+                    // If Span is null it means file level issue and would not have a line. 
```

Comment:
Wouldn't it be better to check the underlying issue and use the new extension method you've created? e.g.

` if (issueViz.Issue.IsFileLevel()) ...`

Here, we're checking a calculated value rather than the original, stable value that is the true indicator of whether an issue is file-level or not.

Or are there other circumstances in which we expect the span to be null?

Reply:
Or perhaps create another `IsFileLevel` extension method that accepts an issue visualization.